packages/keystone/lib/Keystone/index.js

```diff
@@ -157,10 +157,15 @@ module.exports = class Keystone {
       // We do it within the request callback so we can resolve it based on the
       // request info ( like who's logged in right now, etc)
       getCustomAccessControlForUser = memoize(
-        async access => {
+        async (item, args, context, info, access, gqlName) => {
```

Comment:
For consistency with how we call other methods internally, should this use object notation?

```suggestion
        async ({ item, args, context, info, access, gqlName }) => {
```

Reply:
Probably, but this keeps it locally consistent with the other internal functions in`getGraphQlContext`. I'd rather tackle this question as a separate PR, or else we'll end up changing everything.